finalitygadget/finalitygadget_test.go

```diff
@@ -17,21 +17,34 @@ import (
 	"go.uber.org/zap"
 )
 
+type TestFinalityGadget struct {
+	*FinalityGadget
+}
+
+// A helper function for testing only
+func (tfg *TestFinalityGadget) InsertBlocks(blocks []*types.Block) error {
+	return tfg.FinalityGadget.insertBlocks(blocks)
+}
```

Comment:
<img width="745" alt="image" src="https://github.com/user-attachments/assets/191b9cc6-c820-4ae6-82e0-b0793592b886">

we can directly call `insertBlocks` here since it's the same package

Reply:
fixed in https://github.com/babylonlabs-io/finality-gadget/pull/35/commits/3d85e2f7063fff7f52605e7789027c1b46f0741f